src/room.cc

```diff
@@ -628,7 +630,7 @@ void np1secRoom::shrink(std::string leaving_nick) {
           delete old_shrank_session->second;
           session_universe.erase(old_shrank_session->first);
         }
-        session_universe.insert(pair<string, np1secSession*>(action_to_take.bred_session->my_session_id().get_as_stringbuff(), action_to_take.bred_session));
+        session_universe.emplace(std::pair<std::string, np1secSession*>(action_to_take.bred_session->my_session_id().get_as_stringbuff(), action_to_take.bred_session));
```

Comment:
again, emplace instead of insert. I'm not sure if Yawning has don this. But if not, I think when I changed from only storing the pointer to the session instead of session itself, I refrained from using emplace.